src/components/Button/ButtonKitchenSink.tsx

```diff
@@ -241,6 +241,15 @@ function Buttons(): ReactElement {
 				<Button color="gray" variant="icon" mIcon={CloudDownloadIcon} onClick={clickHandler}></Button>
 				<Button color="gray" variant="icon" mIcon={MoreHorizIcon} onClick={openDropdown}></Button>
 			</div>
+
+			<h2>White Icon Buttons</h2>

```

Comment:
I think it's better if we keep all Icon button variants together. This div could go after the last Icon button and just adding "display: inline-block" will make the div appear on the same line.